src/platform/packages/shared/kbn-elastic-agent-utils/src/agent_ingestion_path.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const getIngestionPath = (hasOpenTelemetryFields: boolean) =>
+  hasOpenTelemetryFields ? 'otel_native' : 'classic_apm';
```

Comment:
I'm confused about `classic_apm`, and what `classic` really means. Should it be `elastic_apm` instead?

Reply:
Right. I hadn't seen it. I still find it confusing lol. 🤷